src/cart/index.jsx

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from 'react'
+
+export default class Cart extends Component {
```

Comment:
This is redundant as the PR is about `create a homepage with list of product` not `Cart` so next time please keep the PR clean .

Reply:
I don't got it